### PR TITLE
Add keywords to the npm package

### PR DESCRIPTION
**Pull request by sylwiavargas**
## Summary

This PR adds keywords, description, and a paragraph in the README of our the `inngest` npm package to improve its SEO.

## Checklist
- [ ] ~~Added a [docs PR](https://github.com/inngest/website) that references this PR~~ N/A
- [ ] ~~Added unit/integration tests~~ N/A
- [x] Added changesets if applicable

### CODE REVIEW

File: packages/inngest/README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
   <br/>
   <br/>
   <p>
-    Serverless event-driven queues, background jobs, and scheduled jobs for Typescript.<br />
+    Inngest is the reliability layer for modern applications. It combines durable execution, events, and queues into a zero-infra platform with built-in observability.
+
+    Inngest enables serverless event-driven queues, background jobs, and scheduled jobs for TypeScript.<br />
```
Comment (jpwilliams) [inline packages/inngest/README.md:9]: Is this supposed to be here? The `<br/>` shows up in the doc.

**Review by jpwilliams — COMMENTED** [review]
Yess. Thank you! 🙂 Should help a lot.

File: packages/inngest/README.md (hunk 1 above)
Comment (sylwiavargas) [reply packages/inngest/README.md]: thank you 🫣

**Review by jpwilliams — APPROVED** [review]
Nice! Merge whenever you're happy. :)

**Pull request merged**